src/sentry/apidocs/parameters.py

```diff
@@ -271,3 +271,46 @@ class ReplayParams:
         type=OpenApiTypes.UUID,
         description="""The ID of the replay you'd like to retrieve.""",
     )
+
+
+class IntegrationParams:
+    PROVIDER_KEY = OpenApiParameter(
+        name="providerKey",
+        location="query",
+        required=False,
+        type=str,
+        description="""Specific integration provider to filter by. For example, `slack`. See [Integrations Documentation](/product/integrations/) for an updated list of providers.""",
+    )
+    FEATURES = OpenApiParameter(
+        name="features",
+        location="query",
+        required=False,
+        type=str,
+        many=True,
+        description="""Integration features to filter by. See [Integrations Documentation](/product/integrations/) for an updated list of features. Current available ones are:
+         - alert-rule
+         - chat-unfurl
+         - codeowners
+         - commits
+         - data-forwarding
+         - deployment
+         - enterprise-alert-rule
+         - enterprise-incident-management
+         - incident-management
+         - issue-basic
+         - issue-sync
+         - mobile
+         - serverless
+         - session-replay
+         - stacktrace-link
+         - ticket-rules
+         """,
```

Comment:
Weirdly, multi-line strings have to be formatted like so or else the bullet points end up rendered as code for some reason
```suggestion
        description="""Integration features to filter by. See our [Integration Documentation](/product/integrations/) for an updated list of features. Current available ones are:
 - alert-rule
 - chat-unfurl
 - codeowners
 - commits
 - data-forwarding
 - deployment
 - enterprise-alert-rule
 - enterprise-incident-management
 - incident-management
 - issue-basic
 - issue-sync
 - mobile
 - serverless
 - session-replay
 - stacktrace-link
 - ticket-rules
 """,
```

Reply:
I was wondering. Thanks.